generators/add-extension/index.js

```diff
@@ -110,10 +110,10 @@ class Generator extends Base {
   }
 
   _injectWebExtension(d) {
-    const file = this.destinationPath('phovea.js');
+    const file = this.destinationPath('src/phovea.ts');
```

Comment:
We should have a solution that is backward compatible with the _phovea.js_ files, i.e., that works with TS and JS file types. Otherwise we would get errors with plugins that are not yet updated.

See also #287.

Reply:
Added support for phovea.js files.